packages/react/src/Authenticator.tsx

```diff
@@ -48,7 +52,7 @@ interface AuthenticatorProps {
 
 export function Authenticator ({ children, className = '' }: AuthenticatorProps): JSX.Element {
   return (
-    <AuthCore as='div' className={`w3ui-authenticator ${className}`}>
+    <AuthCore as='div' className={className}>
```

Comment:
So `w3ui-authenticator` means "needs authentication"? Maybe we want to rename to make clearer. Not a blocker for this PR.

Reply:
kinda yes. From the pattern of our components it means "the root element for this widget", and this change makes it so the authenticator doesn't affect the dom if the user has > 0 registered spaces, so it implicitly takes on that meaning. 

I'm not making bigger changes here as we are all in favour of doing less of the "you must give an email to play", but if we revisit this component we'd probably add explicit css classes for "registered" and "not-registered" or similar.